src/App.js

```diff
@@ -1,20 +1,28 @@
-import Navbar from './components/Navbar';
+// import Navbar from './components/Navbar';
+import {UnauthenticatedNavbar, LandlordNavbar, HomeSeekerNavbar} from "./components/Navbar/navbar.js"
 import { Navigate, Route, Routes } from "react-router-dom";
 import './App.css';
 import ListViewPage from "./pages/list-view-page";
 import LandingPage from "./pages/landing-page";
 import PropertyDetailPage from './pages/property-detail-page';
+import MyProperties from "./pages/my-properties-page.js";
+import SavedProperties from "./pages/saved-properties-page.js";
+import { useAuth } from "./context/auth-context";
 
 function App() {
+  const { user, userType} = useAuth();
 
   return (
     <div>
-      <Navbar />
+      {user && userType === "home-seeker" ? <HomeSeekerNavbar /> :
+       user && userType === "landlord" ? <LandlordNavbar /> :<UnauthenticatedNavbar />}
```

Comment:
I don't understand why it has 2 user states if it has the same value.

Reviewing the [auth-context (R8-R9)](https://github.com/andresnb/react-get-that-home/pull/9/files#diff-50d25f7adf348217c540867a8937453cd2a126a2abdf2681a9e4a838a51f9714R8-R9) makes me understand that the user is for testing purposes only, I would recommend simplifying this and using a single variable.

Reply:
This should be better as it only checks for user once and then looks for user type if it exists

 `{user ? userType === "home-seeker" ? <HomeSeekerNavbar /> : <LandlordNavbar /> : <UnauthenticatedNavbar />}`